Appraisals

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+appraise "rails-4" do
+  gem 'actionmailer',  '~> 4.2'
+  gem 'actionpack',    '~> 4.2'
+  gem 'activesupport', '~> 4.2'
+end
+
+appraise "rails-5" do
```

Comment:
Shouldn't we point to 5.2 (or  to be a little more clear? Or, use `~> 5`? That is treated the same as `~> 5.0`. Same comment below.

Reply:
`~> 5.0` will grab the latest stable version of rails 5 to test against. looking at my locks that were generated, it's testing against `5.2.4.2` currently.  i think this is exactly what we want it set to so that every time the CI runs it pulls in the latest stable version of the rails major version.  if you're suggesting changing `rails-5` as the appraisal name to `rails~>5.0`, i don't think that will work well with the appraisal execution `bundle exec appraisal rails~>5.0 rake test`

Reply:
> ~> 5.0 will grab the latest stable version of rails 5 to test against

Agreed. `~> 5` will do the same. (It will _not_ match 6.x!) I was thinking that might be a bit more clear?

(BTW I'm lobbying over in `rubygems` to get this stuff cleaned up. [There should be a clean operator meaning "SemVer match"](https://github.com/rubygems/rubygems/issues/3574) and [the documentation should be clear about how to do that now](https://github.com/rubygems/rubygems/issues/3601)).

Reply:
^ @jebentier ?

Reply:
Shoot sorry, this slipped.  I don't think `~> 5` instead of `~> 5.0` is more clear.  I actually think that `~> 5` is more confusing, given how `~>` is supposed to act, I would expect `~> 5` and `>= 5` to act identically.

Reply:
Ok. It's true that they made an exception for `~> 5` to not work the same as `>= 5`. Further proof that `~>` just wasn't through through very well.

Nick and I both found the `.0` confusing, given that we know it's `.2` right now and will never go backwards from that. So what I was originally suggesting was `~> 5.2` which I read as "the latest stable in the 5 series, which can never be less than the current 5.2".

Reply:
Ahh, I see now. I think I misunderstood your initial comment since it was placed on the appraise line, thinking you were asking to rename the appraisal, rather than change the actual spec definitions.  I'll get this updated in master so the the gems all install `~> 5.2` instead of `~> 5.0`

Reply:
Thank you!